Cargo.toml

```diff
@@ -43,13 +44,15 @@ url = { version = "2.5.2", optional = true }
 
 [dev-dependencies]
 assert_cmd = "2.0.16"
+insta = { version = "1.40.0", features = ["yaml"] }
```

Comment:
I want to propose using https://insta.rs/ for testing -- I have found it to be quite useful (it basically automates the updating of expected output by running `cargo insta review`)